maps/snaxi.dm

```diff
@@ -42,9 +42,10 @@
 
 	snow_theme = TRUE
 	can_enlarge = FALSE
-
+	daynight_cycle = TRUE // TURN IT ON
```

Comment:
And shouldn't this be `STATION_Z`? The two defines have the same value so it would work as it is, but for consistency.